core/src/main/java/io/snyk/plugins/artifactory/scanner/ScannerModule.java

```diff
@@ -59,8 +60,14 @@ public void scanArtifact(@Nonnull RepoPath repoPath) {
     }
 
     if (testResult == null) {
-      LOG.error("Scanning was not successful");
-      return;
+      final String blockOnApiFailurePropertyKey = SCANNER_BLOCK_ON_API_FAILURE.propertyKey();
+      final String blockOnApiFailure = configurationModule.getPropertyOrDefault(SCANNER_BLOCK_ON_API_FAILURE);
+      if ("true".equals(blockOnApiFailure)) {
+        throw new CancelException(format("Artifact '%s' could not be scanned and '%s' is 'true'", repoPath, blockOnApiFailurePropertyKey), 500);
```

Comment:
Could you add "Artifact '%s' could not be scanned because Snyk API is not available"

Reply:
done with 24f71882bf0e155aa1f73d46ebed3f6a0f135381